### ISSUE TITLE
Add mew

### ISSUE THREAD

**Issue by Abrarsultan** [recloudstream/cs-repos#11]
He can u add this site 
https://hindilinks4u.kim/
And some other Hindi sites since there are not many sites on cloud stream which supports Hindi movies and series

**Comment by mabhi9av** [comment]
Yes please! There aren't much alive servers for Hindi audio content. Also this site is great, please devs add this in the app.
